### ISSUE TITLE
loaded manifest data is never actually saved to SaveController

### ISSUE THREAD

**Issue by markmatney** [IIIF/mirador#1041]
`$.DEFAULT_SETTINGS['manifests']` is an Array, but is used like an Object. Since Arrays are Objects, we can set properties on it (like [here](https://github.com/IIIF/mirador/blob/5f020fcf4ad5462143ab3f46e0a2ce11491ac24e/js/src/utils/saveController.js#L246), but calling `JSON.stringify` on it via `SaveController.set` [on the next line](https://github.com/IIIF/mirador/blob/5f020fcf4ad5462143ab3f46e0a2ce11491ac24e/js/src/utils/saveController.js#L247) returns a string that omits all of those properties.

I tried just changing [the value of `'manifests'`](https://github.com/IIIF/mirador/blob/5f020fcf4ad5462143ab3f46e0a2ce11491ac24e/js/src/settings.js#L33) to `{}`, but things break.

**Comment by markmatney** [comment]
_wondering if this is by design and just not documented, and that maybe I missed a discussion about not wanting to save manifest objects in the first place, since they're so darn huge_

**Comment by rsinghal** [comment]
I think this is legacy from 1.0, since I can't remember an exact conversation about it.  But, I think a big part of it is because manifests can be really big, so it's not worth storing the entire thing in a browser's storage.  The URI should be enough because the manifest can be re-requested on reload.

**Comment by markmatney** [comment]
Makes sense, and I agree. In that case, the issues are that (1) the code uses an array (that is always empty) as a key-value store, and (2) the desired behavior (not saving manifest data to localStorage) is achieved in a roundabout way (i.e, by relying on the fact that the key-value pairs of an array object are wiped out when it is stringified). I will update this issue's title - was "loaded manifest data is never actually saved to SaveController".
